.azure-pipelines/gcov.yml

```diff
@@ -46,11 +46,24 @@ jobs:
     ${{ if eq(parameters.pool, 'default') }}:
       vmImage: 'ubuntu-20.04'
 
+  variables:
+    DIFF_COVER_CHECK_THRESHOLD: 0
+    DIFF_COVER_ENABLE: 'true'
+    DIFF_COVER_WORKING_DIRECTORY: $(Build.ArtifactStagingDirectory)/gcov_output/AllMergeReport/
```

Comment:
Currently, the work directory can only be in source folder, or sub-folder of the source folder.